portal-impl/test/unit/com/liferay/portal/minifier/MinifierUtilTest.java

```diff
@@ -31,6 +35,11 @@
 
 	@Before
 	public void setUp() {
+		_minifierEnabled = GetterUtil.getBoolean(
+			PropsUtil.get(PropsKeys.MINIFIER_ENABLED));
+
+		PropsUtil.set(PropsKeys.MINIFIER_ENABLED, "true");
```

Comment:
So this is an integration test after all, not a unit test? 🤔

Reply:
As soon as we saw it fail, we knew it wasn't very unit-ish.

Reply:
Well, the test is a unit test with some "mocking" around for the `Registry` and the `JavaScriptMinifier` to avoid turning it into an integration test. 

Of course if we make it an integration test we won't need that mocking, but then we turn this test into something much heavier and complex to run locally. In this case, as we don't mock a huge number of classes and methods, the benefit of having a unit test is bigger than the cost of mocking.

Also, a lot of mocking in unit tests is a sign of deep coupling in the production code and that should be better addressed in the production code rather than turning everything an integration test, but that's another thing.

So, in this case I would probably go with this solution, though I must admit that it is intriguing why `PropsUtil` works in a unit test at all :roll_eyes: . If we want to investigate that I can try to debug it. If not, we are OK with this change I suppose.

Reply:
Another solution would be mocking the `PropsUtil` class to return `true` for the `MINIFIER_ENABLED` property. That way we don't need to store anything in the test class, but I think both solutions should be equivalent.

Reply:
But how could it be unit but still pass thorugh `PropsValues` -> `PropsUtil.get`? Those were not mocked in the unit test and yet were returning `true` rather than failing... 🤔

Reply:
> So, in this case I would probably go with this solution, though I must admit that it is intriguing why PropsUtil works in a unit test at all 🙄 . If we want to investigate that I can try to debug it. If not, we are OK with this change I suppose.

Yes, this is what's puzzling me too!! ☝️

Reply:
Given that minification is deprecated, hopefully we can just delete the test along with `MinifierUtil` after the next release...

Reply:
> Yes, this is what's puzzling me too!! point_up

Looks like we have an OOB mock for `PropsUtil` :tada: that reads the embedded `portal.properties` (educated guess; I haven't checked it). But @dsanz commented that this pattern (setting a property in a test) is used in other places.